region/crateria/east.json

```diff
@@ -2543,6 +2543,26 @@
                     "It is not necessary to enter the water to clear the rightmost Choot."
                   ]
                 },
+                {
+                  "name": "East Ocean Springball Bounce to the Door with SpaceJump",
+                  "notable": true,
+                  "requires": [
+                    "canTrickyJump",
+                    "canSpaceJumpWaterBounce",
+                    "canSpringBallBounce",
+                    "canMockball",
+                    {"or": [
+                      "canDownGrab",
+                      "canWalljump"
+                    ]}
+                  ],
+                  "note": [
+                    "Build up run speed and then use controlled springball bounces to cross the ocean to the far right ledge, and then use SpaceJump to reach the door.",
+                    "Mockball down the submerged ramp and begin SpringBall bouncing under water using the platforms.",
+                    "Disable SpringBall after the final jump once Samus starts falling again as a way to gain extra speed.",
+                    "It is not necessary to enter the water to kill the right side submerged Choot."
```

Comment:
```suggestion
                    "Disable SpringBall after the final jump once Samus starts falling again as a way to gain extra speed."
```
I feel like this last sentence feels out of place, as the order of the steps jumps around, but also is not really a notable barrier in the strat at this difficulty level - they can easily get out of the water if they do go kill the Choot.

If you do want to keep it, I would suggest reordering some of these sentences